code/implementation/docker/machine/src/main/java/io/cattle/platform/docker/machine/launch/WebsocketProxyLauncher.java

```diff
@@ -51,12 +51,14 @@
 
     protected void prepareConfigFile() throws IOException {
         String config = API_FILTER_PROXY_CONFIG.get();
+        String configContent;
         if (StringUtils.isBlank(config)) {
-            new File(API_FILTER_PROXY_CONFIG_FILE.get()).delete();
+            configContent = "{}";
```

Comment:
@ibuildthecloud what should the service do on Reload, when initially the config is not blank, but later the admin comes in and updates the config to be blank? Should the service continue with the original config since it could not reload any config file, or should it erase its state whenever the config file is not found?